sacro-app/src/main-menu.js

```diff
@@ -85,6 +87,20 @@ const mainMenu = (serverUrl) => {
     });
   }
 
+  let buildDate = "Not defined";
+  const buildDateFile = nodePath.join(__dirname, "../build_date.txt");
```

Comment:
I suspect that build_date.txt is not being included in the installed MSI?

I think you may need to explicitly include it as a resource file in the package.json config, and then load it from the resources dir? Like in find-app-path.js, you may want to try `process.resourcesPath` first, and then load from `__dirname` for dev?

Reply:
Okay, thanks for taking a look at that. I'll give your suggestions a try later. It's proving difficult to debug, because it works fine for me.

Reply:
Right, that should be working fine now :crossed_fingers: 

I've left the log message in, just in case, but can remove it before merge.